src/components/PanelsWrapper.tsx

```diff
@@ -0,0 +1,36 @@
+import { FC, useEffect } from 'react';
+import Panel from '@/components/panel/Panel';
```

Comment:
I think this is imported accidentally. Instead it should come from types.d.ts

Reply:
Actually it is meant to import the Panel component and not the PanelConfig (I updated the name in types.d.ts for the Panel config).